src/3p-frame.js

```diff
@@ -291,7 +291,7 @@ export function getDevelopmentBootstrapBaseUrl(parentWindow, srcFileBasename) {
  */
 function getAdsLocalhost(win) {
   let adsUrl = urls.thirdParty; // local dev with a non-localhost server
-  if (adsUrl.indexOf('ampproject.net') > -1) {
+  if (adsUrl.startsWith('https://3p.ampproject.net')) {
```

Comment:
I don't think this is correct, the domain uses a random subdomain every request. Eg, `d12342.3p.ampproject.net`

Reply:
This one is a breaking change.

Reply:
I wrote this line a year ago to enable local testing. In this case, the URL doesn't have a subdomain.

https://github.com/ampproject/amphtml/blob/c4a663d0ba13d0488c6fe73c55dc8c971ac6ec0d/src/config.js#L37-L38

Reply:
Updated this to be a simple equality check, since this is merely looking for an overridden flag during local testing.